docs/getting-started.md

```diff
@@ -3,33 +3,111 @@ layout: docs
+
+<br>
+
+#### **Option 3. Authenticate to GitHub using a personal access token on your main account**
+
+This option is not recommended as it gives Staticman direct and complete access to your primary GitHub account. Simply, create a personal access token on your primary account.
+
+## Step 2. Deploy Staticman
+
+Read through the [Staticman API config values](https://staticman.net/docs/api) and note the config values you wish to use. At a minimum, you must include a way for Staticman to auth with a git provider, as well as an RSA private key. To generate the RSA private key you can use 
+
+{% highlight bash %}{% raw %}
+openssl genrsa
+{% endraw %}{% endhighlight %}
+
+If you need the newline literals (for example when using JSON configs since multiline strings are not permitted) you can use
+
+{% highlight bash %}{% raw %}
+openssl genrsa | sed '$!s/$/\\n/' | tr -d '\n'
+{% endraw %}{% endhighlight %}
+
+### **Option 1. Deploy to Heroku**
+
+1. Follow <a href="https://heroku.com/deploy?template=https://github.com/eduardoboucas/staticman/tree/master" class="cta">this link</a> to deploy the latest stable Staticman code to Heroku.
+1. You can enter your Staticman API config values as <a href="https://devcenter.heroku.com/articles/config-vars" class="cta">Heroku config variables</a>.
+1. (Optional) Create a fork of Staticman and set up a Heroku pipeline to deploy from the fork. This will help you to keep your instance up to date with the latest Staticman improvements and bug fixes.
+
+<br>
+
+### **Option 2. Deploy to your own infrastructure**
+
+If you prefer to use Docker, check out the [Docker instructions](https://github.com/eduardoboucas/staticman/blob/master/docs/docker.md). Otherwise continue below
+
+1. Clone <a href="https://github.com/eduardoboucas/staticman.git" class="cta">the Staticman repo</a> and ensure `node` and `npm` are installed.
+1. Install the dependencies
+  <br>
+  `npm install`
+1. Create a new config from the sample
+  <br>
+  `cp config.sample.json config.production.json`
+1. Edit the new production config with any values you want, and then start the Staticman server
+  <br>
+  `npm start`
+
+<br>
+
+## Step 3. Create a site configuration file
+
+Staticman will look for a `staticman.yml` file in the root of the repository, where various configuration parameters will be defined. These configuration values are specific to the site. This means one Staticman instance can be used with many websites and users.
+
+You can use the [sample site config file](https://github.com/eduardoboucas/staticman/blob/master/staticman.sample.yml) as a starting point and check the [available site configuration parameters](/docs/configuration) for more information.
+
+## Step 4. Hook up your forms
+
+Forms on your static site should `POST` to:
+
+`{STATICMAN_BASE_URL}/v3/entry/{GIT_PROVIDER}/{GIT_PROVIDER_USERNAME}/{REPO}/{BRANCH}/{property (optional)}`
```

Comment:
It is stated that *Option 1. Authenticate as a GitHub application** is the recommended approach. It uses /v1 however. In that case, will user use the same format for `POST`, i.e., replace `v3` with `v1` or the URL format will also change? This needs some clarification in my humble opinion.

Reply:
Also, is the `GIT_PROVIDER_USERNAME` the name of the Bot? Or name of the repository owner?

Reply:
hmm, actually maybe it is supposed to be `{GIT_PROVIDER/USERNAME}` instead of `{GIT_PROVIDER_USERNAME}` ?

Reply:
GIT_PROVIDER_USERNAME is the name of the repository owner, similar to how github links are structured.